tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/Lambda/DelegateWrapper.cs

```diff
@@ -0,0 +1,132 @@
+// <copyright file="DelegateWrapper.cs" company="Datadog">
```

Comment:
You will need to drop this class in favor of the new mechanism in https://github.com/DataDog/dd-trace-dotnet/pull/4613 once it gets merged.

Reply:
Done.